lib/bolt/transport/ssh/connection.rb

```diff
@@ -309,6 +309,13 @@ def write_remote_executable(dir, file, filename = nil)
           remote_path
         end
 
+        def write_executable_from_content(dest, content, filename = nil)
+          remote_path = "#{dest}/#{filename}"
```

Comment:
`File.join(dest, filename)` is probably more portable.

Reply:
Yeah -- opted for consistency over portability, but I like File.join better and don't think it matters much.

Reply:
Oh, didn't notice above. I'd probably change both.